dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/webrequest/TrackedEntityInstanceCriteria.java

```diff
@@ -186,6 +186,12 @@ public class TrackedEntityInstanceCriteria extends PagingAndSortingCriteriaAdapt
      */
     private boolean includeAllAttributes;
 
+    /**
+     * Indicates whether the Tracked Entity Instance is marked as a Potential
+     * Duplicate.
+     */
```

Comment:
I think the comment here is misleading? This is about a filter parameter, not about the flag of the tei itself - correct?

Reply:
ok i've updated methods javadoc